config/routes.rb

```diff
@@ -17,6 +17,9 @@
   # Deploy
   resource :deploy, only: [:update, :destroy]
 
+  # Grafana dashboards
+  get '/dashboards/:title', to: 'dashboards#show', as: 'title'
```

Comment:
I would just use 
```
resources :dashboards, only: [:show]
```
Putting resources this is translated to `localhost:3000/dashboards/1`, `localhost:3000/dashboards/2`, etc
Handling the id is as simple as having something like
```
class DashboardsController < ApplicationController
  def show
    @id = params[:id].to_i
  end
end
```
And you can use the `@id` in thew views later to decide which dashboards you want to show

Reply:
I know the `resource` pattern but `ID` is not really a proper ID in our case. I don't have any strong opinion on this, both methods  and semantics are accepted by rails community

Reply:
I liked a textual title more than a numerical id, FWIW.